### PR TITLE
Deleted Container Explanation

### PR DESCRIPTION
**Pull request by cindylay**
## Category

- [x ] Content fix
- [ ] New article

## What's in this Pull Request?

1. This adds an explanation to the Deleted Container section in the Consuming Tenant Admin article.
2. Updated "deleted container collection" to "Recycle Bin"

### CODE REVIEW

File: docs/embedded/concepts/app-concepts/containertypes.md [hunk 1]
```diff
@@ -22,17 +22,34 @@ The permission mapping between a ContainerTypeID-AppID pair determines the opera
 > [!TIP]
 > To learn more about application architecture, see [Application Architecture](./app-architecture.md).
 
-## SharePoint Embedded Trial Container Types
+
+## TODO: Introduce two types
```
Comment (andrewconnell) [inline docs/embedded/concepts/app-concepts/containertypes.md:26]: This doesn't look complete. Please complete this section before submitting the PR.

File: docs/embedded/concepts/app-concepts/containertypes.md [hunk 2]
```diff
@@ -22,17 +22,34 @@ The permission mapping between a ContainerTypeID-AppID pair determines the opera
...
+
+### SharePoint Embedded Trial Container Types
 
 To aid customers with the development of applications to be used with SharePoint Embedded, we're allowing the creation of a Trial Container Type. Customers are allowed one Trial Container Type at any time in their tenant. The Trial Container Type won't be linked to billing and can only be consumed by the Partner tenant.
 
 > [!NOTE]
 > The Trial Container Type is the only Container Type that is restricted to the Partner (creating) tenant and cannot be shared with other consuming tenants.
 
+### Create a Trial Container Type: Visual Studio Code Extension
+You can create a Trial Container Type using the SharePoint Embedded Visual Studio Code Extension [here](https://marketplace.visualstudio.com/items?itemName=SharepointEmbedded.ms-sharepoint-embedded-vscode-extension)
```
Comment (andrewconnell) [inline docs/embedded/concepts/app-concepts/containertypes.md:43]: MS Docs recommend linking the text of the target, not words like 'click this link' or 'here'.

File: docs/embedded/concepts/app-concepts/containertypes.md [hunk 3]
```diff
@@ -45,3 +62,7 @@ To delete a trial Container Type, you must remove all Containers of that type fi
 Customers can choose to create a new Trial Container Type at any time, if the above steps are performed to remove any pre-existing Trial Container Type, are performed. The Trial Container Type is valid for up to 30 days but can be removed at any time within the 30-day window.
 
 Also note that it isn't possible* to convert a Trial Container Type and/or the associated Containers to a Standard Container Type.
+
+
+#TODO:
```
Comment (andrewconnell) [inline docs/embedded/concepts/app-concepts/containertypes.md:67]: Incompelte

File: docs/embedded/tutorials/register-api.md [hunk 4]
```diff
@@ -0,0 +1,139 @@
+---
+title: Register File Storage Container Type Application Permissions
+description: Register the Container Type
+ms.date: 02/26/2024
+ms.localizationpriority: high
+---
+
+# Register File Storage Container Type Application Permissions
+
+TODO: Explain Registration API purpose
```
Comment (andrewconnell) [inline docs/embedded/tutorials/register-api.md:10]: incomplete

File: docs/embedded/tutorials/register-api.md [hunk 5]
```diff
@@ -0,0 +1,139 @@
...
+- called in Consuming Tenant
+- Can be called 1+ times
+- container.selected scope adds entry in enterprise applications
+
+
+This registers the `fileStorageContainerType` within the Consuming Tenant.  
+
+The calling app's identity from token will be used to determine the owning tenant of the app and validate the ownership of the container type. Only the owning app can register container type app permissions.
+
+>>  Note this API must be called with a cert-based access token
```
Comment (andrewconnell) [inline docs/embedded/tutorials/register-api.md:20]: incorrect ms node docs format

File: docs/embedded/tutorials/register-api.md [hunk 6]
```diff
@@ -0,0 +1,139 @@
...
+TODO: Expand on this certificate based auth
```
Comment (andrewconnell) [inline docs/embedded/tutorials/register-api.md:21]: incomplete

File: docs/embedded/tutorials/register-api.md [hunk 7]
```diff
@@ -0,0 +1,139 @@
...
+
+## Container.Selected Scope 
+
+This API uses the Container.Selected scope. To call this API you must configure this scope in your App manifest.  
+
+TODO: This is NOT a Graph API, clearly state this is a SharePoint API
```
Comment (andrewconnell) [inline docs/embedded/tutorials/register-api.md:27]: incomplete

File: docs/embedded/tutorials/register-api.md [hunk 8]
```diff
@@ -0,0 +1,139 @@
...
+>>  Note this API must be called with a cert-based access token
+TODO: Expand on this certificate based auth
+
+## Container.Selected Scope 
+
+This API uses the Container.Selected scope. To call this API you must configure this scope in your App manifest.  
+
+TODO: This is NOT a Graph API, clearly state this is a SharePoint API 
+
+>>  Note that other SharePoint Embedded Graph APIs run with the FileStorageContainer.Selected Scope on Microsoft Graph
```
Comment (andrewconnell) [inline docs/embedded/tutorials/register-api.md:29]: incorrect msdocs alert format

File: docs/embedded/tutorials/register-api.md [hunk 9]
```diff
@@ -0,0 +1,139 @@
...
+TODO: This is NOT a Graph API, clearly state this is a SharePoint API 
+
+>>  Note that other SharePoint Embedded Graph APIs run with the FileStorageContainer.Selected Scope on Microsoft Graph 
+
+
+### Configure your App Manifest
+
+In your app manifest you will need to: 
+
+Select Manage > Manifest from the left-hand navigation. Locate the property requiredResourceAccess and edit it so it looks like the following JSON:
```
Comment (andrewconnell) [inline docs/embedded/tutorials/register-api.md:36]: markdown format issues: bold navigation items, and mark inline code as code

File: docs/embedded/tutorials/register-api.md [hunk 10]
```diff
@@ -0,0 +1,139 @@
...
+| Container.Selected | Application | Allows the application to utilize the file storage container platform to manage containers without a signed in user. The specific file storage containers and the permissions granted to them will be configured in Microsoft 365 by the developer of each container type. |
+
+> You must configure the `Container.Selected` scope in your App manifest
+
+## HTTP request
+
+```http
+PUT {RootSiteUrl}/_api/v2.1/storageContainerTypes/{containerTypeId}/applicationPermissions
+```
+TODO: Explain RootSiteUrl,
```
Comment (andrewconnell) [inline docs/embedded/tutorials/register-api.md:82]: incomplete

File: docs/embedded/tutorials/register-ct.md [hunk 11]
```diff
@@ -1,13 +1,62 @@
...
 ---
 
-# Registering a Container Type
+# Create and Register a Container Type
 
-## Create a new Container Type for your app
+This article will explain how to create and register a billable [Container Type](../concepts/app-concepts/containertypes.md) in SharePoint Embedded.
+
+**Pre-requisites:**
+* You must have admin credentials to a M365 developer tenant
```
Comment (andrewconnell) [inline docs/embedded/tutorials/register-ct.md:13]: bullets should have breaklines around them

File: docs/embedded/tutorials/register-ct.md [hunk 12]
```diff
@@ -1,13 +1,62 @@
...
+* You must register your application in [Microsoft Entra ID](https://learn.microsoft.com/en-us/training/modules/sharepoint-embedded-setup/5-exercise-setup-configure-sharepoint-embedded)
```
Comment (andrewconnell) [inline docs/embedded/tutorials/register-ct.md:14]: don't use absolute links or locales when referencing pages in learn.Microsoft.com

File: docs/embedded/tutorials/register-ct.md [hunk 13]
```diff
@@ -1,13 +1,62 @@
...
+## Set up an Azure Subscription Id, Resource Group, and Region
+
+### 1. Set up Azure Subscription Id
+Login to Azure Portal with the admin credentials of your tenant
+
+
+(TODO: add the extra existing Sub id)
+
+Navigate to [Subscriptions](https://portal.azure.com/#view/Microsoft_Azure_Billing/SubscriptionsBladeV2)
+![alt text](azuresubscription.png)
```
Comment (andrewconnell) [inline docs/embedded/tutorials/register-ct.md:27]: invalid image alt text

File: docs/embedded/tutorials/register-ct.md [hunk 14]
```diff
@@ -1,13 +1,62 @@
...
+
+If you do not have an existing subscription set-up, click add
+![alt text](suboption.png)
```
Comment (andrewconnell) [inline docs/embedded/tutorials/register-ct.md:30]: invalid image alt text

File: docs/embedded/tutorials/register-ct.md [hunk 15]
```diff
@@ -1,13 +1,62 @@
...
+
+Choose a subscription plan. You will be prompted to add in credit card information that will be billed when you create File Storage Containers. This will set you up with an Azure Subscription Id.
+
+### 2. Set up a Resource Group and Region
+Navigate to [Resource Groups](https://portal.azure.com/#view/HubsExtension/BrowseResourceGroups) in Azure Portal.
+![alt text](resource-group.png)
```
Comment (andrewconnell) [inline docs/embedded/tutorials/register-ct.md:36]: invalid image alt text

File: docs/embedded/tutorials/register-ct.md [hunk 16]
```diff
@@ -1,13 +1,62 @@
...
+
+Click "Create" to create a resource group and name it something appropiate to your application. Then click Review + Create
+![alt text](c-rg.png)
```
Comment (andrewconnell) [inline docs/embedded/tutorials/register-ct.md:39]: invalid image alt text

File: docs/embedded/tutorials/register-ct.md [hunk 17]
```diff
@@ -1,13 +1,62 @@
...
+
+Make sure that:
+* The appropiate Azure Subcription is selected for the resource group
+* The appropiate physical region is selected. For ex. (US) East US
+
+>> Note: The Region Group does NOT require your Containers to live in that specified region. Your content is not restricted to the specified region
```
Comment (andrewconnell) [inline docs/embedded/tutorials/register-ct.md:45]: invalid msdocs alert format

File: docs/embedded/tutorials/register-ct.md [hunk 18]
```diff
@@ -1,13 +1,62 @@
...
+
+**TODO: Need to explain Region and Resource Group in relation to SharePoint Embedded
```
Comment (andrewconnell) [inline docs/embedded/tutorials/register-ct.md:47]: incomplete

File: docs/embedded/tutorials/register-ct.md [hunk 19]
```diff
@@ -1,13 +1,62 @@
...
+**
+
+Click Review + Create and your validation should pass if the inputs are correct.
+
+Now you have all the necessary resources to create a Billable Container Type.
+
+<br></br>
```
Comment (andrewconnell) [inline docs/embedded/tutorials/register-ct.md:54]: don't use html in markdown unless aboslutely necessary (like breaklines within tables). these should be removed

File: docs/embedded/tutorials/register-ct.md [hunk 20]
```diff
@@ -31,7 +86,10 @@ New-SPOContainerType -ContainerTypeName {{ContainerTypeName}} -OwningApplication
 
 The cmdlet outputs your new `ContainerTypeId`, save this for later use.
 
-## Grant admin consent for your app
+### 3. Grant admin consent for your app
+
+TODO: Take this out of the article and put it into the register CT article....
```
Comment (andrewconnell) [inline docs/embedded/tutorials/register-ct.md:91]: incomplete

File: docs/embedded/tutorials/register-ct.md [hunk 21]
```diff
@@ -40,9 +98,22 @@ You can grant admin consent to an app by visiting the following link with the
...
+## Register Your Container Type in the Consuming Tenant
+
+### 4. Register the new Container Type
+
+There are two options to register your new Container Type and provide application permissions.
+
+### Calling the Register Container Type API
+
+See the API documentation for Register File Storage Container Type Application Permissions [here](register-api.md)
+* Please Note that you will need to use a cert-based access token to make this API call.
```
Comment (andrewconnell) [inline docs/embedded/tutorials/register-ct.md:110]: should this be a real msdocs alert?

**Review by andrewconnell — CHANGES_REQUESTED** [review]
This appears to be a work in progress PR... if that's correct, please use set to a draft PR to indicate it's not ready for review. I'll do this now as that's what it seems with the numerous "todo" comments in the content.

Open PR's communicate that it's ready to be reviewed & merged. Thanks!

**Comment by cindylay** [comment]
Thanks for the feedback @andrewconnell , I realize I've duplicated Container Type content here from #9562 . I've added the CTA content changes to #9562 and will implement the other Container Type feedback listed here as well.

**Pull request closed without merging**